### DISCUSSION TITLE
Default to "webrtc" instead of "mse"?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#4907] [Ask A Question] [answered]
When I initially go to a specific camera the default mode is "mse". If I change it to "webrtc" then come back to the camera is remains "webrtc", which is great. Is there a way to have it default to "webrtc" initially?

Thank you.

**Comment by NickM-27** [comment, marked as answer]
No, the default is the more compatible mse but it will stick after you set it.
